app/templates/jinja2/dashboard/volunteer/signup_players.html

```diff
@@ -49,6 +49,53 @@
       {% endfor %}
       </tbody>
     </table>
+    <table class="table table-striped">
+      <thead>
+      <tr>
+        <th scope="col">Self registration</th>
+        <th scope="col">Total registration</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td scope="col">{{ player_count - sum([loc.signupinvite_set.filter(used_at__isnull=False).count() for loc in locations]) }}</td>
```

Comment:
This should be done in the backend and passed in.

Reply:
Also, this throws an error locally as it's not syntactically valid for some reason. Please fix this.